ochre/Analysis.py

```diff
@@ -379,6 +380,14 @@ def calculate_metrics(results=None, results_file=None, dwelling=None, metrics_ve
             metrics['Total Hot Water Delivered (kWh)'] = \
                 results['Hot Water Delivered (W)'].sum(skipna=False) / 1000 * hr_per_step
 
+    # EV metrics
+    if "EV SOC (-)" in results:
+        metrics["Average EV SOC (-)"] = results["EV SOC (-)"].mean(skipna=False)
+    if "EV Unmet Load (kW)" in results:
```

Comment:
Can we address the simple things in https://github.com/NREL/OCHRE/issues/144 before we merge? Units change at least, if we're going to merge this metric in it should probably be correct.

The assumed 100% SoC at the end I dealt with in a different project by running a baseline and doing a difference, I'm fine with pushing that off to later.